src/lib.rs

```diff
@@ -62,12 +62,17 @@
 #![warn(missing_docs)]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 
+use core::fmt;
+
 use bitflags::bitflags;
 
-macro_rules! wait_for {
+macro_rules! spin_unwrap {
```

Comment:
I'm not sure about the `unwrap` name. Maybe something like `retry_until_ok`?

Reply:
Done 👍